scipy/stats/tests/test_distributions.py

```diff
@@ -679,6 +679,79 @@ def test_cdf(self):
                                5.06649130e-01, 8.41442111e-01], atol=1e-6)
 
 
+class TestKappa4(TestCase):
+    def test_cdf_genpareto(self):
+        # h = 1 and k != 0 is generalized Pareto
+        x = [0.0, 0.1, 0.2, 0.5]
+        h = 1.0
+        for k in [-1.9, -1.0, -0.5, -0.2, -0.1, 0.1, 0.2, 0.5, 1.0,
+                  1.9]:
+            vals = stats.kappa4.cdf(x, h, k)
+            # shape parameter is opposite what is expected
+            vals_comp = stats.genpareto.cdf(x, -k)
+            assert_allclose(vals, vals_comp)
+
+    def test_cdf_genextreme(self):
+        # h = 0 and k != 0 is generalized extreme value
+        x = [-0.5, -0.2, -0.1, 0.0, 0.1, 0.2, 0.5]
+        h = 0.0
+        for k in [-1.9, -1.0, -0.5, -0.2, -0.1, 0.1, 0.2, 0.5, 1.0,
+                  1.9]:
+            vals = stats.kappa4.cdf(x, h, k)
+            vals_comp = stats.genextreme.cdf(x, k)
+            assert_allclose(vals, vals_comp)
+
+    def test_cdf_genlogistic(self):
+        # h = -1.0 and k != 0 is generalized logistic
+
+        # Couldn't get the generalized logistic test to work, I think
+        # mostly because of how to establish the "c" parameter for the
+        # general logistic distribution.
+        pass
```

Comment:
Skimming http://etd.lsu.edu/docs/available/etd-05182004-144851/, (found from quick googling), it seems that `h=-1` and `k=0` matches genlogistic with `c=1`  (cf Eq. (13) in the reference above)